src/stdlib.cpp

```diff
@@ -117,12 +121,12 @@ auto type(const CallContext& ctx) -> Value {
     return v.type();
 }
 
-auto assert(const CallContext& ctx) -> Value {
+auto assert_lua(const CallContext& ctx) -> Vallist {
     auto v = ctx.arguments().get(0);
     auto message = ctx.arguments().get(1);
 
-    if (v) {
-        return v;
+    if (bool(v)) {
```

Comment:
Not sure if the `bool(v)` is needed. Maybe just `v` would also work. But I'm not sure because the `operator bool()` is explicit.

Reply:
tested it, with just `v`, it works.